CMakeLists.txt

```diff
@@ -133,14 +133,15 @@ SET (ORION_LIBS
     rest
     jsonParse
     jsonParseV2
-    apiTypesV2
+    rest
```

Comment:
Not a problem.
Could be better, as we seem to have a cross dependency between libraries.
However, it's no problem having the linker go over a library twice.